src/services/wallets.service.js

```diff
@@ -6,7 +6,26 @@
  */
 
 const initializedWallets = new Map();
+/**
+ * @typedef TxProposalConfig
+ * @property {string} id Proposal identifier on the Atomic Swap Service
+ * @property {string} password Password to access it
+ */
+
+/**
+ * A map of all the proposals for a wallet.
+ * The keys are the proposal ids
+ * @typedef {Map<string,TxProposalConfig>} WalletListenedProposals
+ */
+
+/**
+ * A map of the initialized wallets and their listened proposals.
+ * The keys are the wallet-ids
+ * @type {Map<string,WalletListenedProposals>}
+ */
+const walletListenedProposals = new Map();
```

Comment:
Why don't you add this to the `atomic-swap.service` file? I feel it belongs there.

Reply:
Refactored it on [f7960e4](https://github.com/HathorNetwork/hathor-wallet-headless/pull/279/commits/f7960e4fd4e54f23e85ac6114279c19034fb7703). That's a good place for this variable to be.